prisma/schema.prisma

```diff
@@ -85,7 +86,7 @@ model JobPost {
   scrubsProvided  Boolean
   experience      String
   location        String
-  shifts          String
+  shiftsTime          String
```

Comment:
wouldn't this effect some of the frontend components? or break anything as previously the shifts entry was a string and now we changed the name to shiftsTime